src/modules/managedata.js

```diff
@@ -35,18 +35,18 @@ class Todo {
     });
   }
 
- // clearCompleted() {
-  //  this.setTodos(this.todos.filter((singleTodo) => singleTodo.completed === false));
-  //}
-
-  //complete(index) {
-    //this.todos = this.todos.map((singleTodo) => {
-      //if (singleTodo.index === index) {
-        //singleTodo.completed = !singleTodo.completed;
-     // }
-      //return singleTodo;
-    //});
-  //}
+  clearCompleted() {
+    this.setTodos(this.todos.filter((singleTodo) => singleTodo.completed === false));
+  }
```

Comment:
- [ ] [Optional] When you clear all completed tasks, kindly reorder the value of `index` key to start from the **_length of the array + 1_**. You can take a cue from the `this.removeTodo` method that you have in your class `Todo`. 👍